pages/linux/eix.md

```diff
@@ -0,0 +1,27 @@
+# eix
+
+> Utilities for searching local Gentoo packages.
+
+- Synchronize local cache (should be ran after installation/updating packages):
+
+`eix-update`
```

Comment:
typo? This looks like a different command to me

Reply:
Ah, you're right. I've changed this to be a comment in the command description, like in [apt-get](https://github.com/tldr-pages/tldr/edit/master/pages/linux/apt-get.md).